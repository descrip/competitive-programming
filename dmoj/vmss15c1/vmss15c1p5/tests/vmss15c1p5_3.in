9 1000 945
APPL 617 3 31
candy 740 18 28
internet 179 8 3
reddit 295 20 22
stang 297 27 5
stromae 700 7 3
thisisaname 268 8 1
yokoono 190 5 8
zxcvb 53 7 10
