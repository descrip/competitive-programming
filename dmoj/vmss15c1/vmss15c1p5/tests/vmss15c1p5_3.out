105072
APPL 16
candy 0
internet 0
reddit 0
stang 0
stromae 136
thisisaname 0
yokoono 0
zxcvb 0
